Formula/pypy.rb

```diff
@@ -78,42 +80,27 @@ def install
     python = buildpath/"bootstrap/bin/pypy"
 
     cd "pypy/goal" do
-      system python, buildpath/"rpython/bin/rpython",
-             "-Ojit", "--shared", "--cc", ENV.cc, "--verbose",
-             "--make-jobs", ENV.make_jobs, "targetpypystandalone.py"
-
-      with_env(PYTHONPATH: buildpath) do
-        system "./pypy-c", buildpath/"lib_pypy/pypy_tools/build_cffi_imports.py"
-      end
+      system python, "../../rpython/bin/rpython", "--opt", "jit",
+                                                  "--cc", ENV.cc,
+                                                  "--make-jobs", ENV.make_jobs,
+                                                  "--shared",
+                                                  "--verbose"
     end
 
+    system python, "pypy/tool/release/package.py", "--archive-name", "pypy",
+                                                   "--targetdir", ".",
+                                                   "--no-embedded-dependencies",
+                                                   "--no-keep-debug",
+                                                   "--no-make-portable"
     libexec.mkpath
-    cd "pypy/tool/release" do
-      package_args = %w[--archive-name pypy --targetdir . --no-make-portable --no-embedded-dependencies]
-      system python, "package.py", *package_args
-      system "tar", "-C", libexec.to_s, "--strip-components", "1", "-xf", "pypy.tar.bz2"
-    end
-
-    (libexec/"lib").install libexec/"bin/#{shared_library("libpypy-c")}"
-    if OS.mac?
-      MachO::Tools.change_install_name("#{libexec}/bin/pypy",
-                                       "@rpath/libpypy-c.dylib",
-                                       "#{libexec}/lib/libpypy-c.dylib")
-    end
```

Comment:
I think we can avoid moving this library as it is installed in `libexec/bin` rather than `bin`. May need someone familiar with macOS RPATH behavior to confirm this doesn't have any issues. On Linux, keeping `libexec/bin/libpypy-c.so` and creating symlink in `lib` is similar to Arch Linux package (https://github.com/archlinux/svntogit-community/blob/packages/pypy/trunk/PKGBUILD#L42-L49)